cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -276,8 +276,7 @@ private void checkMonolithicConfigurationException() {
       if (this.templatesProject == null)
         LOG.warn(monolithicConfigurationMessage + "{} ", ConfigurationFinder.findTemplatesLocation());
       else
-        LOG.warn(monolithicConfigurationMessage + "{} ", this.templatesProject);
-      LOG.warn(e.getMessage());
+        LOG.warn(monolithicConfigurationMessage + "{} {}", this.templatesProject, e.getMessage());
```

Comment:
Please remove the concatenation using +. We've discussed this with @maybeec already.
Better would be to start a string and use {} instead.